src/main/java/bit/javaoop/CEO.java

```diff
@@ -0,0 +1,31 @@
+package bit.javaoop;
+
+import java.util.List;
+
+public class CEO{
+    List<Manager> listofManagers;
+    String name;
+    double salary;
+
+    public CEO(String name, double salary) {
+        this.name = name;
+        this.salary=salary;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public void HireManager(Manager manager) {listofManagers.add(manager);}
+
+    @Override
+    public String toString() {
+        String str=name+"- CEO \n";
+        for(Manager m:listofManagers) str+=m.toString();
```

Comment:
Tutaj wspomnę o dobrodziejstwach IDE'łki (IntelliJ'a) 
- Raz ctrl + alt + L  formatuje kod warto tak sobie to co chwilę kliknąć nabrać takiego nawyku
- Dwa:
![image](https://user-images.githubusercontent.com/18233796/37936026-3cef7c6a-3154-11e8-8f04-a15894007a6c.png)
+= nam się podkreśla dajemy na niego kursor i czytamy co tam się dzieje klikamy alt + enter patrzymy czy proponuje nam jakieś rozwiązanie ( IDEłka jest w tych czasach naprawdę mądrym stworkiem):D
W tym wypadku proponuje zamianę Stringa na StringBuildera i ma rację bo robiąc taką konkatenację w pętli kompilator nie jest w stanie ogarnąć żeby podstawić StringBuildera więc w efekcie tworzy nam się tyle Stringów ilu mamy managerów każdy kolejny dłuższy.
A przecież tak naprawdę interesuje nas tylko ostatni:)